client/src/pages/Launch.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import logoDark from "../assets/ddLogoDark.svg";
+import styled from "styled-components/macro";
+
+const Wrapper = styled.div`
+  height: 100vh;
+  display: grid;
+  grid-template-rows: 40% 60%;
+  grid-template-areas:
+    "title"
+    "logo";
+`;
+
+const Title = styled.h1`
+  grid-area: title;
+  place-self: end center;
+  padding-bottom: 2rem;
+  font-size: 3rem;
+  color: var(--white);
```

Comment:
I would rename this variable to something less specific

Reply:
Replaced with primary and secondary 🚀